ssds/__init__.py

```diff
@@ -156,3 +158,49 @@ def sync(submission_id: str, src: SSDS, dst: SSDS) -> Generator[str, None, None]
                 f"submission_id='{submission_id}' "
                 f"src='{src}' "
                 f"dst='{dst}'")
+
+def release(submission_id: str, src: SSDS, dst: SSDS, transfers: List[Tuple[str, str]]) -> dict:
```

Comment:
Should `submission_id: str` be `submission_id: Optional[str]`?

Reply:
Nope! This method explicitly relies on `submission_id`.